UKHomeOffice/nmsw-ui
# NMSW-1059 Change more actions from radio form to links

# Ticket



----

## AC

![image](https://github.com/UKHomeOffice/nmsw-ui/assets/724554/6a3d2982-2a8a-4979-b3af-e57f90eb0c3b)


----

## To test

- create a declaration until you can get to the UPT results page
- click the link for adding another FAL 6
- > you should be taken to the passenger question and be able to continue to selecting no or uploading a new fal 6
- go back to UPT results page
- click the link for save and come back later
- > you should be taken to the task list page and passengers should be `in progress`
- go back to the UPT results page
- click the button for finish
- > you should be taken to the task list page and passengers should be `complete`

----

## Notes
